supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1079,6 +1079,22 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rulebook]]
+==== image:images/yes.png[yes] rulebook (noun)
+*Description*: A _rulebook_ is a YAML file containing a list of rules, event source definitions, and hosts to run against. 
+Rulebooks are used in Event-Driven Ansible. When using the term _rulebook_ without the Ansible prefix, use lowercase _r_.
+
+Examples:
+
+* Use a rulebook in Ansible.
+* Use an Ansible Playbook.
```

Comment:
```suggestion
* Use an Ansible Rulebook.
```

I think you meant Rulebook, not Playbook? :)

Reply:
Oh gosh, so sorry! Forgot to wake up this morning 😅  I'll correct it!

Reply:
OK that should be updated! Apologies again and thanks for catching that @bredamc :)